slurm-installation/overview.rst

```diff
@@ -248,3 +246,59 @@ After setting the node up, to bring it back you need to run a Juju *action*:
    juju-compute-GsLk    up   infinite      1   idle juju-01ab62-3
    configurator*     inact   infinite      1   idle juju-01ab62-1
 
+
+CentOS7 Deploys on LXD clouds
+-----------------------------
+
+You need to manually create an LXD image for CentOS7 in order to deploy it with
+Juju.
+
+First step is to download a configuration file describing the image:
+
+.. code-block:: bash
+
+    $ wget https://raw.githubusercontent.com/lxc/lxc-ci/master/images/centos.yaml
+
+Juju needs two extra packages (`sudo` and `openssh-server`) that are not in the
+base image. You need to manually add them in the `packages` section of the yaml
+file. The first `set` of packages in the file should then be:
+
+.. code-block:: bash
+
+    packages:
+      manager: yum
+      update: true
+      cleanup: true
+      sets:
+      - packages:
+        - cronie
+        - cronie-noanacron
+        - curl
+        - dhclient
+        - initscripts
+        - openssh-clients
+        - passwd
+        - policycoreutils
+        - rootfiles
+        - rsyslog
+        - vim-minimal
+        - sudo
+        - openssh-server
+        action: install
+
+Now we need to install `distrobuilder` and generate the image:
+
+.. code-block:: bash
+
+    $ sudo snap install distrobuilder --classic
+    $ sudo distrobuilder build-lxd image.yaml -o image.architecture=x86_64 -o image.release=7 -o image.variant=cloud
```

Comment:
The file referenced above is `centos.yaml`, use the same filename here.

Reply:
copy and paste failure here. Thanks for noticing it!